include/audi/invert_map.hpp

```diff
@@ -59,7 +59,7 @@ taylor_map operator-(const taylor_map &A, const taylor_map &B)
 
 // This is the "multiplication" operator.
 template <typename T>
-taylor_map operator*(const T &c, const taylor_map &A)
+inline taylor_map operator*(const T &c, const taylor_map &A)
```

Comment:
I'm not sure if you also need inline when the function is a template. @darioizzo do you know?

Reply:
yes you need it, function templates are not automatically ``inline``.

Reply:
Okay thanks!